core/tests/test_utils.py

```diff
@@ -73,11 +73,26 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Sets the content, status code and headers that the urlfetch mock
+        should return with.
+
+        Args:
+            content: str. The content to return in subsequent calls to the
+            urlfetch mock.
```

Comment:
Yes, @pranavsid98 is right -- this line needs to be indented by 4 additional spaces.